src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationHandlers.cs

```diff
@@ -1526,9 +1559,15 @@ ProviderTypes.Twitter     or ProviderTypes.Weibo         or ProviderTypes.Zoom
                 ProviderTypes.Vimeo => (string?) context.UserInfoResponse?["uri"] is string uri &&
                     uri.StartsWith("/users/", StringComparison.Ordinal) ? uri["/users/".Length..] : null,
 
+                // VK ID returns the user identifier as a custom "user/user_id" node:
+                ProviderTypes.VkId => (string?) context.UserInfoResponse?["user"]?["user_id"],
```

Comment:
Same here, remove that and merge it with the `ProviderTypes.Amazon or ProviderTypes.HubSpot or ProviderTypes.StackExchange or ProviderTypes.Typeform` case.